content/en/docs/armory-admin/armory-agent/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Armory Agent for Kubernetes"
+linkTitle: "Armory Agent"
+weight: 4
+description: >
+  How the Armory Agent works
+---
+
+The Armory Agent is a lightweight service that connects back to Spinnaker's Clouddriver service and streams changes from the Kubernetes clusters it is configured to watch.
+
+> The Agent connects back to a new port opened on Clouddriver using gRPC.
+
+
+## Topologies
+
+### Spinnaker service mode
+
+In this mode, the Agent is installed as a new Spinnaker service (`spin-kubesvc`) and can be configured like other services.
+
+![Spinnaker service mode](/images/armory-agent/in-cluster-mode.png)
+
+If you provision clusters automatically, the Agent can dynamically reload accounts when `kubesvc.yaml` changes. You could, for example, configure accounts in a `configMap` mounting to `/opt/spinnaker/config/kubesvc-local.yaml`.  The Agent reflects `configMap` within seconds after [etcd](https://etcd.io/) sync.
+
+### Infrastructure mode
+
+In infrastructure mode, multiple Agent deployments handle different groups of Kubernetes clusters. Each deployment is configured separately.
+
+![Infra mode](/images/armory-agent/agent-infra-mode.png)
+
+> Account name must still be unique across all your infrastructure. Clouddriver will reject new accounts with a name that matches a different cluster.
+
+### Agent mode
+
+In this mode, the Agent acts as a piece of infrastructure. It is provisioned with a `serviceAccount` properly scoped to what you want Spinnaker to do. The cluster will disappear from Spinnaker after a grace period when the cluster is deleted or when the Agent is stopped.
+
+![Agent mode](/images/armory-agent/agent-mode.png)
+
+
+## Security
```

Comment:
I'd move this after `Communication` or even as a subtopic like `Information sent by the Agent` since it's the communication being secured, right?